birdhouse/deployment/fix-geoserver-data-dir-perm

```diff
@@ -31,4 +31,4 @@ docker run --rm --name fix-geoserver-data-dir-perm \
     --env FIRST_RUN_ONLY \
     "${BASH_IMAGE}" \
     bash -xc 'if [ -z "$FIRST_RUN_ONLY" -o ! -f /datadir/global.xml ]; \
-    then chown -R 1000:10001 /datadir; else echo "No execute."; fi'
+    then chown -R 2000:2000 /datadir; else echo "No execute."; fi'
```

Comment:
Maybe make this a parameter for the script, e.g.: `GEOSERVER_DATA_OWNER`?
If latest birdhouse is used but `GEOSERVER_VERSION` is pinned to an older version, this would break its configuration.

Reply:
If you're going to do that @tlvu can you please leave a comment at the top of this file that says that it's deprecated and telling the user that it's no longer needed for newer versions of geoserver (and let them know which versions it's fixed for)

That way we'll know that it's no longer needed and why.

Reply:
> That way we'll know that it's no longer needed and why.

Yes will do.  I still have a few problems with this new GeoServer and our existing data to sort out.  The upgrade did not work seamlessly.  Will update this and probably other configs to get the upgrade to work correctly at the same time.